test-cases/gutenberg/writing-flow/multiline-components.md

```diff
@@ -15,6 +15,7 @@ Test the next steps on:
 - On Android, pressing enter multiple times in the content of a Pullquote or Quote block causes unintended behavior ([#27690](https://github.com/WordPress/gutenberg/issues/27690))
 - On Android, there is sometimes a loss of lines when entering multiple lines. ([#29861](https://github.com/WordPress/gutenberg/issues/29861))
 - There is sometimes inconsistent HTML surrounding newlines in the Pullquote and Quote Block.  ([#1396](https://github.com/WordPress/gutenberg/issues/1396))
+- On Android, pressing enter multiple times in a Pullquote removes lines as reported in [#27690]. Once that happens hitting backspace to delete characters adds characters to words being deleted ([#3495](https://github.com/wordpress-mobile/gutenberg-mobile/issues/3495))
```

Comment:
<img width="844" alt="Screen Shot 2021-05-12 at 21 32 19" src="https://user-images.githubusercontent.com/1898325/118064731-95bc0f80-b369-11eb-9caa-08f3ba5062ad.png">

This #27690 link doesn't seem to render, I think it might be because it's a Gutenberg issue (https://github.com/WordPress/gutenberg/issues/27690) and so it's not in the `wordpress-mobile` org, so GitHub can't find it.